lexicons/com/atproto/repo/checkMissingBlobs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.repo.listMissingBlobs",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Returns a list of missing blobs for the requesting account. Intended to be used in the account migration flow.",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 1000,
+            "default": 500
+          },
+          "cursor": { "type": "string" }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["cids"],
+          "properties": {
+            "cursor": { "type": "string" },
+            "cids": {
+              "type": "array",
+              "items": { "type": "string", "format": "cid" }
```

Comment:
I don't know if it is easy or worth it, but if this returned blob objects (eg, with size, mimetype, etc, in addition to CID), might be more ergonomic and helpful for humans. and machines could do things like compute total size of missing objects.

Reply:
hmmm that's not a bad idea 🤔 down to take a stab at it

And to answer the prev question (which GH isn't letting me respond to for some reason), this is not accessible to external accounts, but purged blobs would not show up in the output

Reply:
Updated it to the `blob` type (similar to output of `uploadBlob`) which contains all that info 👍